### ISSUE TITLE
UI: Export ACL List rules returns [object Object]

### ISSUE THREAD

**Issue by OlegChuev** [apache/cloudstack#8256]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.18
~~~

##### SUMMARY
Export of ACL Rules with persisted tags returns an unreadable output in csv file.
<img width="702" alt="image" src="https://github.com/apache/cloudstack/assets/91337563/e74f3f72-2b34-41e7-8a63-36337d58d67e">

##### STEPS TO REPRODUCE

~~~
0) Login to ACS, navigate to the VPC tab in the left panel and select/create any VPC, navigate to the VPC view page;
1) Navigate to the Network ACL lists page;
2) Select any non-default / create ACL rule;
3) Assign tags for the following non-default ACL rule;
4) Click 'Export ACLs' button;
~~~

##### EXPECTED RESULTS

~~~
Tags are represented in a readable way
~~~

##### ACTUAL RESULTS

~~~
Tags are represented as a [Object object]
~~~

**Comment by DaanHoogland** [comment]
@OlegChuev do you see chance to test #8258 ?

**Comment by DaanHoogland** [comment]
btw, as this is not an API functionality but a UI export, I think an import will be problamatic as well.

**Comment by OlegChuev** [comment]
> @OlegChuev do you see chance to test #8258 ?

I tested and left one comment; could you please take a look? I think it's worth sanitizing all special characters in the tags string to prevent any possible issues.

Overall LGTM

**Comment by DaanHoogland** [comment]
@OlegChuev 

> I tested and left one comment;

can you link me to your comment? I don see it in #8258

**Comment by OlegChuev** [comment]
> @OlegChuev
> 
> > I tested and left one comment;
> 
> can you link me to your comment? I don see it in #8258

Sure: https://github.com/apache/cloudstack/pull/8258/files#r1401076643

**Comment by shwstppr** [comment]
@DaanHoogland cc @OlegChuev can this be closed as #8258 is merged or do we need further improvements?
